File: src/main/java/org/taktik/connector/business/chapterIV/validators/Chapter4XmlValidator.java
package org.taktik.connector.business.chapterIV.validators;

import org.taktik.connector.business.chapterIV.exception.ChapterIVBusinessConnectorException;
import org.taktik.connector.technical.exception.TechnicalConnectorException;
import java.io.Serializable;

public interface Chapter4XmlValidator extends Serializable {
   void validate(Object var1) throws ChapterIVBusinessConnectorException, TechnicalConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/chapterIV/validators/Chapter4XmlValidator.kt
package org.taktik.connector.business.chapterIV.validators

import org.taktik.connector.business.chapterIV.exception.ChapterIVBusinessConnectorException
import org.taktik.connector.technical.exception.TechnicalConnectorException
import java.io.Serializable

interface Chapter4XmlValidator : Serializable {
    @Throws(ChapterIVBusinessConnectorException::class, TechnicalConnectorException::class)
    fun validate(xmlObject: Any?)
}
